neurobooth_os/tasks/DSC.py

```diff
@@ -386,8 +386,11 @@ def nextTrial(self):
         out_fname = f"{self.subj_id}_{self.task_name}_outcomes{self.rep}.csv"
         df_res.to_csv(self.path_out + res_fname)
         df_out.to_csv(self.path_out + out_fname)
-        if len(self.task_files):
+        if len(self.task_files) >= 1:
             self.task_files = self.task_files[-1] + f", {res_fname}, {out_fname}" + "}"
```

Comment:
I'm having trouble understanding this without the mental context of the surrounding code. Not clear to me how this works without a loop... but if we test in staging with a few repeats and it looks good then thumbs up from me.